gensim/models/wrappers/dtmmodel.py

```diff
@@ -93,6 +93,9 @@ def __init__(
             lencorpus = sum(1 for _ in corpus)
         if lencorpus == 0:
             raise ValueError("cannot compute DTM over an empty corpus")
+        if any([i == 0 for i in [len(text) for text in corpus.get_texts()]]) and model == "fixed:
```

Comment:
Wouldn't it be better to do `if model == "fixed" and any([i == 0 for i in [len(text) for text in corpus.get_texts()]]):`
so that it won't bother checking the condition if it isn't in `fixed` mode?